src/compare.js

```diff
@@ -0,0 +1,41 @@
+import fs from 'fs';
+import fsExtra from 'fs-extra';
+import {PNG} from 'pngjs';
+import pixelmatch from 'pixelmatch';
+import { createHash } from 'crypto';
+
+const compare = (snapshot, latest, output, update, threshold = 0.01) => new Promise((resolve, reject) => {
+    const latestImg = PNG.sync.read(fs.readFileSync(latest));
+
+    if(update || !fs.existsSync(snapshot)) {
```

Comment:
The code previously had a space between `if` and `(`. I'd keep the style as consistent as possible with what was there previously.

(Might be worth it to add prettier to the project later on so that formatting becomes a non-issue)